ci-operator/jobs/openshift-kni/baremetal-deploy/openshift-kni-baremetal-deploy-master-presubmits.yaml

```diff
@@ -0,0 +1,72 @@
+presubmits:
+  openshift-kni/baremetal-deploy:
+  - agent: kubernetes
+    always_run: true
```

Comment:
I think this is the line causing the complaint in the config since the addition of run_if_changed

Reply:
Ah yes, the message is pretty explanatory "is set to always run but also declares run_if_changed targets, which are mutually exclusive"